[WIP] Add Security to Ubuntu Bare Metal Guide

alexhersh opened the pull request.

Commit:
Update Calico-Ubuntu Guide
+ Use v1.1.2 assets
+ Add Secure API Authentication
+ Reference Updated calico-kubernetes config

tomdee commented on docs/getting-started-guides/ubuntu-calico.md:
```diff
@@ -78,8 +88,8 @@ We'll use the `kubelet` to bootstrap the Kubernetes master processes as containe
 wget https://github.com/kubernetes/kubernetes/releases/download/v1.1.1/kubernetes.tar.gz
```
How about getting them a different way:

```
sudo wget -N -P /usr/bin http://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/amd64/kubectl
sudo wget -N -P /usr/bin http://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/amd64/kubelet
sudo chmod +x /usr/bin/kubelet /usr/bin/kubectl
```

```
```

tomdee commented on docs/getting-started-guides/ubuntu-calico.md:
```diff
@@ -58,13 +58,23 @@ On each Node:
+sudo mv apiserver-key.pem /etc/kubernetes/ssl/
+
+# Set Permissions
+sudo chmod 600 /etc/kubernetes/ssl/*-key.pem
```
Why are you using a wildcard here? It only matches one file so seems dangerous...

tomdee commented on docs/getting-started-guides/ubuntu-calico.md:
````diff
@@ -88,7 +98,12 @@ sudo cp -f kubernetes/server/bin/kubectl /usr/bin
+On your master, download the `calico-kubernetes` repository, which contains the necessary configuration for this guide.
+
 ```
+wget https://github.com/projectcalico/calico-kubernetes/archive/master.tar.gz
````
I think it's better to download the files as they are required rather than grabbing the whole repo. They can be placed in the right directory as they are downloaded.

Commit:
Fixing issues
+ Condense wgets
+ Add worker config
+ kube-proxy.service -> kube-proxy.manifest
